content/40.inequality.md

```diff
@@ -148,7 +155,9 @@ A preprint [@doi:10.1101/2020.04.05.20054502] provided observational evidence th
 Although lack of individual-level exposure data and the impossibility of randomization make it difficult to elucidate the exact causal mechanism, this finding would be consistent with similar findings for all-cause mortality (e.g., [@doi:10.1073/pnas.1803222115]).
 Exposure to air pollution is associated with both poverty (e.g., [@doi:10.3390/ijerph15061114]) and chronic inflammation [@doi:10.1016/j.neubiorev.2018.06.002].
 Other outcomes of environmental racism, such as the proximity of abandoned uranium mines to Navajo land, can also cause respiratory illnesses and other health issues [@url:https://journalhosting.ucalgary.ca/index.php/jisd/article/view/70753/54416].
-Similarly, preliminary findings indicate that nutritional status (e.g., vitamin D deficiency [@doi:10.1210/clinem/dgaa733]) may be associated with COVID-19 outcomes, and reduced access to grocery stores and fresh food often co-occurs with environmental racism [@url:https://journalhosting.ucalgary.ca/index.php/jisd/article/view/70753/54416; @doi:10.1056/NEJMp2021264].
+Similarly, preliminary findings indicate that nutritional status (e.g., vitamin D deficiency [@doi:10.1210/clinem/dgaa733; @doi:10.3390/nu12041181; @doi:10.3390/nu12051466; @arxiv:2102.02250v1]]) may be associated with COVID-19 outcomes, and reduced access to grocery stores and fresh food often co-occurs with environmental racism [@url:https://journalhosting.ucalgary.ca/index.php/jisd/article/view/70753/54416; @doi:10.1056/NEJMp2021264]. 
```

Comment:
I added #930 with the nutraceuticals manuscript in mind, but it may be relevant here as well.  @RLordan what's your opinion on this preprint?

Reply:
Thanks @agitter, I just had a quick look. The quality of the 11 studies included is very variable. Individually, from what I remember the majority of them conclude there is an association with mortality. Interesting that this meta concludes the opposite. I wouldn't bet the contents of my apartment on the findings of this preprint just yet. The same issues of study quality are making it difficult to definitively determine associations between vitamin D and disease severity too.

Reply:
@agitter @rando2 
On a related note, we might want to take a closer look at this paper for this section. The UK biobank is not a good platform to make determinations on associations between vitamin D and covid outcomes, but thats not what caught my attention, it was the lack of association between socioeconomic and cardiometabolic outcomes and Covid that caught my eye. 
 https://doi.org/10.1093/pubmed/fdaa095